### PR TITLE
Clean up the README

### PR DESCRIPTION
**Pull request by introwit**
- Anything else a PHP dev can do for you? UI, UX & docs are something I can help with :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,29 @@
+<h1 align="center"> GitPoint </h1> <br>
 <p align="center">
   <a href="https://gitpoint.co/">
     <img alt="GitPoint" title="GitPoint" src="http://i.imgur.com/VShxJHs.png" width="450">
   </a>
 </p>
 
 <p align="center">
-  GitHub for iOS. Built with React Native.  </p>
+  GitHub for iOS. <a href="https://github.com/gitpoint/git-point/issues/2#issuecomment-316261018">Android comming soon </a> <br>
```
Comment (lex111) [inline README.md]: > comming
typo

**Comment by lex111** [comment]
@introwit thank you for your contribution, although I do not really like the alignment in the center (it does not seem that it looks more neat), I would leave everything as is, except for the information about the version for Android.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
+</p>
+
+<p align="center">
+  <a href="
+https://travis-ci.org/gitpoint/git-point"><img src="https://img.shields.io/travis/gitpoint/git-point.svg?style=flat-square" alt="Build Status"></a>
+  <a href="#contributors"><img src="https://img.shields.io/badge/all_contributors-18-orange.svg?style=flat-square" alt="All Contributors"></a>
```
Comment (alejandronanez) [inline README.md]: Please, keep contributors out of html tags. It will break the ability to auto update contributors counter when we do `contributors:add`

Comment (alejandronanez) [inline README.md]: That being said, I think all badges should be out of any html tags.

Comment (andrewda) [inline README.md]: Personally I think everything should be out of HTML (except things that really need it like center tags). All images and link should be markdown for consistency and readability's sake.

Comment (alejandronanez) [inline README.md]: Agree!

**Comment by introwit** [comment]
@andrewda sorry, was away from my laptop on sunday. Since there are multiple suggestions & you already made a commit, let me know the things left or the things I need to do, will do it right now :) If you want to close this, I have no issues with that :)

**Comment by andrewda** [comment]
@introwit No problem, only thing that still needs to be fixed is using markdown for the badges for reasons listed in the review. It would be great if you could take a look at that :smile:

**Comment by introwit** [comment]
@andrewda done :+1:

**Pull request merged**